source/funkin/options/type/NumOption.hx

```diff
@@ -0,0 +1,51 @@
+package funkin.options.type;
+
+import flixel.effects.FlxFlicker;
+
+class NumOption extends OptionType {
+	public var selectCallback:Float->Void;
+
+	private var __text:Alphabet;
+	private var __number:Alphabet;
+
+	public var min:Float;
+	public var max:Float;
+
+	public var currentSelection:Float;
+	public var changeVal:Float;
+
+	var optionName:String;
+
+	public var text(get, set):String;
+	private function get_text() {return __text.text;}
+	private function set_text(v:String) {return __text.text = v;}
+
+	public function new(text:String, desc:String, min:Float, max:Float, changeVal:Float, optionName:String, ?selectCallback:Float->Void = null) {
+		super(desc);
+		this.selectCallback = selectCallback;
+		this.min = min;
+		this.max = max;
+		if(Reflect.field(Options, optionName) != null)
```

Comment:
This here is supposed to be like `parent, optionName`
Because of like custom settings